werf.yaml

```diff
@@ -298,6 +298,7 @@ git:
   - testing
   - go.mod
   - go.sum
+  - .golangci.yaml
```

Comment:
Why is it in pr?
If it is important, make a new one pls.

Reply:
Got it, fixed.